src/components/dashboard/main-content/course-enrollments/course-cards/tests/BaseCourseCard.test.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-no-constructed-context-values */
```

Comment:
Similar question as earlier for being consistent about how we're handling this rule in test files.